interbtc-spec/docs/source/spec/issue.rst

```diff
@@ -185,7 +186,7 @@ Specification
 *Events*
 
 * :ref:`executeIssueEvent`
-* :ref:`issueAmountChangeEvent`
+* :ref:`issueAmountChangeEvent` (conditional)
```

Comment:
How do we want to represent conditional events? cc: @nud3l @sander2 @shonfeder

Reply:
Made an update